documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -6,6 +6,9 @@ Release Notes
 ======
 * The multi-version client API would not propagate errors that occurred when creating databases on external clients. This could result in a invalid memory accesses. `(PR #5221) <https://github.com/apple/foundationdb/pull/5221>`_
 * Fixed a race between the multi-version client connecting to a cluster and destroying the database that could cause an assertion failure. `(PR #5221) <https://github.com/apple/foundationdb/pull/5221>`_
+* Added Mako latency measurements. `(PR #5255) <https://github.com/apple/foundationdb/pull/5255>`_
+* Fixed a bug when passing InconsistentSnapshotOnly. `(PR #5228) <https://github.com/apple/foundationdb/pull/5228>`_
```

Comment:
Release notes for bugs are usually structured to describe the impact of the bug being fixed. For example, see the two release notes in lines 7 and 8. Can you update this to follow that pattern?

Reply:
Done.